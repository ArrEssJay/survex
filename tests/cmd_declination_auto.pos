( Easting, Northing, Altitude )
(410600.00, 5282000.00,  1234.00 ) 1
(410600.00, 5282000.00,  1234.00 ) a.1
(410593.61, 5282007.62,  1232.95 ) a.2
(410593.61, 5282008.84,  1233.08 ) a.3
(410593.61, 5282007.62,  1232.95 ) b.1
(410593.61, 5282007.62,  1231.95 ) b.2
(410593.61, 5282007.62,  1231.95 ) c.1
(410593.61, 5282007.62,  1230.95 ) c.2
(410594.29, 5282007.18,  1229.13 ) c.3
(410593.61, 5282007.62,  1230.95 ) d.1
(410596.58, 5282107.57,  1230.95 ) d.2
