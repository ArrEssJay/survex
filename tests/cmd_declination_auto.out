./cmd_declination_auto.svx:11: warning: No survey date specified - using 0 for magnetic declination
 1 2 10.00 320 -06
./cmd_declination_auto.svx:27: warning: No survey date specified - using 0 for magnetic declination
 2 3 2.0 123 -66

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 12 survey stations, joined by 11 legs.
There are 0 loops.
Total length of survey legs =  165.23m ( 165.23m adjusted)
Total plan length of survey legs =  161.98m
Total vertical length of survey legs =    5.00m
Vertical range = 4.87m (from a.1 at 1234.00m to c.3 at 1229.13m)
North-South range = 107.61m (from d.2 at 5282107.61m to a.1 at 5282000.00m)
East-West range = 6.27m (from a.1 at 410600.00m to d.1 at 410593.73m)
   5 1-nodes.
   4 2-nodes.
   3 3-nodes.
There were 2 warning(s).
